src-java/network-topology/network-storm-topology/src/main/java/org/openkilda/wfm/topology/network/service/NetworkIslService.java

```diff
@@ -176,6 +176,7 @@ public void bfdEnableDisable(IslReference reference, IslBfdFlagUpdated payload)
             bfdManager.enable(carrier);
         } else {
             bfdManager.disable(carrier);
+            bfdManager.disableUnhurriedPoll(carrier);
```

Comment:
I have doubts regarding this call here - `bfdManager.disable(carrier)` will trigger BFD removal process and at the end, it will trigger `bfdManager.disableUnhurriedPoll` inside IslFsm in managed manner... This call here is a "fire and forget" approach... it should work in most cases, but in some cases, it can lead to extremely unexpected results.

Can we try to not put it here?

Reply:
You are right. This call has been removed.